components/brave_ads/core/internal/targeting/geographical/subdivision/README.md

```diff
@@ -1,4 +1,4 @@
-# Subdivision
+# Subdivision targeting
```

Comment:
nit: `Subdivision Targeting`

Reply:
ah ok, can we remove the word targeting then please

Reply:
oh, sorry I misread the comment, the current file is actually in `targeting/geographical/subdivision` directory. So it should be Targeting. I will fix

Reply:
Was this fixed, as I still see lowercase ` t` for targeting?